### ISSUE TITLE
Weird issue

### ISSUE THREAD

**Issue by mdlieber99** [TheLastBen/fast-stable-diffusion#2423]
You've helped with a number of issues that I have had, but now I have a weird one.  It's with someone else's note (kohya_ss) for extracting lora's but I thought you might be able to help since you seem to know your way around these things.  I'm happy to make a contribution (ko-fi?) for helping with this.  

I'm running https://colab.research.google.com/github/panguin6010/kohya_ss_google_colab/blob/master/kohya_ss_colab.ipynb?pli=1#scrollTo=IzS3hvuTtTqW

and then going to 'utilities' and then to 'extract lora'.  I've gotten this to work in the past, but doing the same thing now just gets an error after it churns for a bit.  below is my inputs and then the outputs.  I know this isn't your problem/notebook, but I'd be very grateful if you could point me to a solution.

here is what I have filled in:

Finetuned model:
/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/RevcowPerfLev10_step_35000.ckpt

Stable diffusion base model:
/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/PerfectLevel10.safetensors

Save to:
/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora/revcow.safetensors

everything else left as default. here is the output:

python3 "networks/extract_lora_from_models.py" --save_precision float --save_to "/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora/revcow.safetensors" --model_org "/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/PerfectLevel10.safetensors" --model_tuned "/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/RevcowPerfLev10_step_35000.ckpt" --dim 128 --device cuda --conv_dim 128
2023-07-31 17:15:28.478035: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations: AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-07-31 17:15:30.487057: E tensorflow/stream_executor/cuda/cuda_blas.cc:2981] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2023-07-31 17:15:34.330257: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64
2023-07-31 17:15:34.330786: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64
2023-07-31 17:15:34.330923: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
loading SD model : /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/PerfectLevel10.safetensors
loading u-net:
loading vae:
loading text encoder:
loading SD model : /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/RevcowPerfLev10_step_35000.ckpt
loading u-net:
loading vae:
Killed

any ideas? help much appreciated!

**Comment by TheLastBen** [comment]
not enough RAM, try using high-RAM runtime

**Comment by mdlieber99** [comment]
Of course - it's my default with your notebook, but I didn't check it there.  Thanks!
